pkg/cli/cmds/server.go

```diff
@@ -212,6 +213,11 @@ var ServerFlags = []cli.Flag{
 		Destination: &ServerConfig.FlannelBackend,
 		Value:       "vxlan",
 	},
+	&cli.BoolFlag{
+		Name:        "multi-cluster-cidr",
+		Usage:       "(networking) Enable multiClusterCIDR (v1.26 Alpha feature)",
```

Comment:
Since this is a 1.26 alpha, we should mark this as experimental until it goes GA in upstream